### PR TITLE
Install PostgreSQL 9.5

### PR DESCRIPTION
**Pull request by gildegoma**
* [x] wait for PostgreSQL [9.5.0 GA release](http://www.postgresql.org/developer/roadmap/), which ETA is still for end 2015
* [x] decide how we'll deal with PostGIS integration (move to 2.2 could be an interesting option, see the [support matrix](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS))
  * PostGIS 2.1 supports from 9.1 to 9.4, but 9.5 is for the moment not accepted/planned
  * PostGIS 2.2 supports from 9.1 to **9.5**

---

**Open Questions:**

* should this change be back-ported to `precise-stable` branch as well ?
* I guess that the above question depends on when `Trusty` will be out of beta? (Attention the new LTS [Xenial Xerus is coming](https://wiki.ubuntu.com/XenialXerus) ;-)
 The following points are worth to be considered when deciding which default version and additional versions should be shipped:
 * `9.1` was selected as default version by the time, because it was the default version on Precise packaging. 
 * On Trusty, the default/official version is now `9.3`.
 * 9.0 has been EOLed in September 2015, but 9.1 is still maintained.

Related to travis-ci/travis-ci#4264

### CODE REVIEW

File: cookbooks/postgresql/recipes/postgis.rb [hunk 1]
```diff
@@ -18,5 +18,5 @@
 # At the moment, the same PostGIS *single* version is installed for all PostgreSQL instances
 #
 ([node['postgresql']['default_version']] + node['postgresql']['alternate_versions']).each do |pg_version|
-  package "postgresql-#{pg_version}-postgis-#{node['postgresql']['postgis_version']}"
+  package "postgresql-#{pg_version}-postgis-#{node['postgresql']['postgis_version']}" unless pg_version == '9.5'
```
Comment (gildegoma) [inline cookbooks/postgresql/recipes/postgis.rb]: The unless skipping condition should be removed when 9.5.0 is out.

**Comment by phlipper** [comment]
PostgreSQL 9.5 has been [officially released](http://www.postgresql.org/docs/9.5/static/release-9-5.html) as of 2016-01-07.

**Comment by gildegoma** [comment]
@phlipper yes indeed :)

We have to clarify a few more things before merging. For instance, PostGIS 2.1 (which is the version currently available in Travis build environments) is not planned (so far) to be supported with 9.5. Moving to PostGIS 2.2 must be evaluated then...

That said, I am absolutely looking forward to shipping this "asap" ;-)

**Comment by BanzaiMan** [comment]
Is PostGIS 2.2 out?

**Comment by gildegoma** [comment]
@BanzaiMan Yes, PostGIS 2.2 is out for 4 months, and the awesome people from DebianGIS+UbuntuGIS have recently released the related APT packages. I updated this feature branch, and things should be ready to be merged and tried on Trusty :-)

**Comment by phlipper** [comment]
Any update on this?

**Comment by gildegoma** [comment]
@BanzaiMan @meatballhat OK to merge?

**Comment by meatballhat** [comment]
@gildegoma @phlipper Sorry about the yuuuuge delay.  Getting this merged shortly, and hopefully shipping updated `group: edge` image(s) in the next week.

**Comment by gildegoma** [comment]
@meatballhat ❤️❤️❤️

**Comment by deepj** [comment]
When can we expect PostgreSQL 9.5 between addons on Travis CI?

**Pull request merged**